autoshop/analytics/processor.py

```diff
@@ -0,0 +1,4 @@
+def related_website(request):
```

Comment:
This should be in `autoshop` app or in `core` app as its a site wide context processor.
Also rename it to `site_processor`.